File: regestrationform/app/src/main/java/com/example/regestrationform/MainActivity.java
package com.example.regestrationform;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    EditText ed1,ed2,ed3,ed4;
    Button btnOk;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ed1 = findViewById(R.id.name);
        ed2 = findViewById(R.id.address);
        ed3 = findViewById(R.id.phone);
        ed4 = findViewById(R.id.email);

        btnOk = findViewById(R.id.btnk);
        btnOk.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

                String name = ed1.getText().toString();
                String address = ed2.getText().toString();
                String phone = ed3.getText().toString();
                String email = ed4.getText().toString();


                String txt =   "Your Name is "  + "\t"  +  name  +  "\n" +
                        "Your Address is " + "\t"  + address + "\n" +
                        "Your Phone No is " + "\t"  + phone + "\n" +
                        "Your Email ID is " + "\t"  + email + "\n";

                Toast.makeText(MainActivity.this,txt.toString(), Toast.LENGTH_LONG).show();

            }
        });
    }
}

File: MyApplication/app/src/main/java/com/example/myapplication/MainActivity.kt
package com.example.myapplication

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    lateinit var edittext:EditText
    lateinit var edittext2:EditText
    lateinit var button: Button

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        edittext=findViewById(R.id.edittext)
        edittext2=findViewById(R.id.edittext2)
        button=findViewById(R.id.button)
        button.setOnClickListener{
            if (edittext.text.toString().isEmpty()){
                Toast.makeText(applicationContext,"Please enter your name",Toast.LENGTH_SHORT).show()
            }
            else if (edittext2.text.toString().isEmpty()){
                Toast.makeText(applicationContext,"please enter your pasward",Toast.LENGTH_SHORT).show()

            }
        }
    }
}